specification/azurefleet/AzureFleet.Management/examples/2024-05-01-preview/Fleets_Get.json

```diff
@@ -314,11 +314,6 @@
                 "extensionsTimeBudget": "mbhjahtdygwgyszdwjtvlvtgchdwil"
               },
               "licenseType": "v",
-              "priority": "Regular",
```

Comment:
why we have this change?

Reply:
These are forbidden fields which we are blocking for customers. Hence I removed them. Earlier there was no way to hide them as we were using useref. But now we don't want them as we define priority on our side and don't want customers to be confused.

Reply:
Did you do breaking changes review of this?